pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -89,19 +93,25 @@ func (s *Service) GetAPIServerDNSName() (string, error) {
 func (s *Service) DeleteLoadbalancers() error {
 	s.scope.V(2).Info("Deleting load balancers")
 
-	// Get default api server name.
-	elbName := GenerateELBName(s.scope.Name(), infrav1.APIServerRoleTagValue)
-
-	// Describe and delete if exists.
-	if _, err := s.describeClassicELB(elbName); err != nil {
-		if IsNotFound(err) {
-			return nil
-		}
+	elbs, err := s.listOwnedELBs()
+	if err != nil {
 		return err
 	}
 
-	if err := s.deleteClassicELBAndWait(elbName); err != nil {
```

Comment:
Can we remove the entire `deleteClassicELBAndWait` function - I don't think it's used anywhere else?